tests/connection_tests.rb

```diff
@@ -91,5 +91,9 @@
     end
   end
 
+  test('initialization') do
+    Excon::Connection.new # This should not raise
```

Comment:
I think this is failing the tests because it's not `true`. How about:
```suggestion
    !Excon::Connection.new.nil? # This should not raise
```

Reply:
I can't get green tests locally even without my changes, so let's try that and we'll see